### ISSUE TITLE
Color constrast violation not found in Edge & IE

### ISSUE THREAD

**Issue by 40thieves** [dequelabs/axe-core#570]
We have a known color contrast violation on [sharelatex.com](https://www.sharelatex.com) (we're working on it!).

Running `axe.run` in Chrome 61 on macOS returns the color contrast violation as expected:

![screen shot 2017-10-13 at 10 39 13](https://user-images.githubusercontent.com/424411/31540377-66c17124-b003-11e7-8516-65dbc7bf623c.png)

However running `axe.run` in Edge 15 on Windows 10 (via Browserstack) does not return the color contrast violation:

![screen shot 2017-10-13 at 10 34 34](https://user-images.githubusercontent.com/424411/31540506-c55e9216-b003-11e7-97b8-ed4c2b37207f.png)

Thanks for creating such a useful tool! 😃

**Comment by 40thieves** [comment]
Also just fyi here's the script I used to run the tests:

```js
var s = document.createElement('script')
s.type = 'text/javascript'
s.async = true
s.src = 'https://unpkg.com/axe-core/axe.min.js'
s.onload = function() {
  axe.run(function(err, results) {
    console.log(err, results)
  })
}
document.body.appendChild(s)
```

**Comment by WilcoFiers** [comment]
Page has been updated. Closing the issue. Apologies for not replying to this issue sooner. This one seems to have slipped, unfortunately.

**Comment by 40thieves** [comment]
Right, sorry the a better url would now be https://www.overleaf.com/about.

But I just re-tested and it now seems to work, so happy to keep this closed. Thanks again 😄
